server/authz/policy.rego

```diff
@@ -620,3 +620,20 @@ allow {
   team_role(subject, object.team_id) == [admin, maintainer][_]
   action == [read, write][_]
 }
+
+# Global admins and maintainers can read and write bootstrap packages.
+allow {
+  object.type == "mdm_apple_bootstrap_package"
+  subject.global_role == [admin, maintainer][_]
+  action == [read, write][_]
+}
+
+# Team admins and maintainers can read and write bootstrap packages on their teams.
+allow {
+  not is_null(object.team_id)
+  object.team_id != 0
```

Comment:
Is this necessary? (Other similar rules do not have this.)

Reply:
to be completely honest, I copied it from the rules for profiles. Theoretically makes sense to prevent team admins/maintainers from modifying a "no team" bootstrap package, I'm not 100% sure if it's necessary.